src/publishTestAssets.proj

```diff
@@ -1,9 +1,9 @@
 <Project DefaultTargets="PublishTestAssets">
```

Comment:
Why are this file an others like `sentohelix.proj` moved out of the `eng` dir?

Reply:
Those two project files either implicitly or explicitly import the Directory.Build.props/targets from corefx. Means those are very repo specific so it's just natural to move them into src. I left the other files in eng as those can be shared over-time with the other parts easily.